dashboard/test/unit/test_cbroker.js

```diff
@@ -253,5 +254,25 @@ suite('cbroker', function () {
 
     });
 
+    test('should_filter_region_list_and_remove_disabled_region', function () {
```

Comment:
Just some comments here.

- A disabled region will have correct data in the CB, although it was obsoleted.
- Why do you add in the expected the 'Region5' without data?